libraries/chain/finality_core.cpp

```diff
@@ -0,0 +1,302 @@
+#include <eosio/chain/finality_core.hpp>
+#include <eosio/chain/block_header.hpp>
+
+#warning Remove undef NDEBUG for assert before RC
+//Undefine NDEBUG to enable assertions in CICD.
+#undef NDEBUG
+#include <cassert>
```

Comment:
I would think this would need to be the first file included as the files above this could have already included `<cassert>`.

Reply:
Thanks. You are correct. Will change.

Reply:
Moved to the very top.